src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
I don't think we want to hard-code the "latest" dependencies in a file like this.  It would be better to have a special project.json like this that contains a placeholder value for each, like:
```
  "Microsoft.NETCore.TestHost": "1.0.0-rc3-$USEVERSION$",
```
Then if we ask to build with latest versions, an MSBuild task would copy that to some known location (in bin) and replace $VERSION with a value we specify on the command line (e.g. /p:UseVersion="*" or /p:UseVersion="23729").  This will make it much more flexible because we can ask to test with any specific version we want, not just "the latest on myget".  I know my comments make the scope of this change bigger than you'd planned, but I really think what I'm proposing is the right design for this feature.

I thought @dagood initially had a task that did something similar.  It's possible we broke it along the way.